test/filters/test_convolution.cpp

```diff
@@ -3155,6 +3158,268 @@ main (int argc, char** argv)
   (*input) (61,47).intensity = 100;
   (*input) (62,47).intensity = 98;
   (*input) (63,47).intensity = 96;
+
+  // filter
+  PointCloud<PointXYZI>::Ptr output;
```

Comment:
From SunBlack:
> Define Eigen::ArrayXf filter(7); inline before, so it is easier to see that these are the initiale values.

So I defined all variables near the initialization, which is not a bad idea because there is so much data in the first test.

Reply:
How is this related to my comment?

Reply:
I got that wrong. I would still do without it because I know that the double bracket `>>' at the end of `auto {in,out}put = pcl::make_shared<PointCloud<PointXYZI>>();` can cause problems with some compiler versions. Yes, you can prevent this by putting a space in between. But I don't find this solution very robust, because some IDEs can delete them during automatic code formatting.

Reply:
> can cause problems with some compiler versions

That was true for old compilers (pre C++11). Since we support only C++14 (or newer), it's not a concern for a conformant compiler

Reply:
Ok